website/src/graphql/dailp/queries.graphql

```diff
@@ -531,67 +451,24 @@ mutation CurateWordAudio($input: CurateWordAudioInput!) {
   }
 }
 
+fragment BookmarkedDocument on AnnotatedDoc {
+  id
+  title
+  slug
+  bookmarkedOn {
+    formattedDate
+  }
+}
+
```

Comment:
This fragment could survive with only `slug` and `bookmarkedOn`, since `slug` is our key for documents in GraphCache. That said, keeping title and id feels saner to me.